src/app/InteractionModelEngine.cpp

```diff
@@ -411,6 +412,21 @@ Status InteractionModelEngine::OnInvokeCommandRequest(Messaging::ExchangeContext
         ChipLogProgress(InteractionModel, "no resource for Invoke interaction");
         return Status::Busy;
     }
+    CHIP_FAULT_INJECT(FaultInjection::kFault_BatchCommandIdmTcHelper1,
+                      commandHandler->TestOnlyTcIdm1_3FaultInjection(
+                          apExchangeContext, std::move(aPayload), aIsTimedInvoke, /* oneReponsePerMessage = */ true,
```

Comment:
usage of the last 3 bools make it seem like they should just be some enum ... we call TFF, FTF and FFT.